N Queens.py

```diff
@@ -0,0 +1,68 @@
+# TC : O(n!)
+# SC : O(n)
```

Comment:
SC is O(n^2)

Reply:
Ohh yes. My bad, I didnt notice. Will change it.